src/lib/element-renderer.ts

```diff
@@ -39,5 +39,5 @@ export interface ElementRenderer {
    *
    * @param c The element _class_ to render styles for.
    */
-  renderStyles(c: Constructor<HTMLElement>): Iterator<string>;
+  renderStyles(c: HTMLElement): Iterator<string>;
```

Comment:
ooh... I think we can remove this since we removed flattened mode. This was to output Shady DOM scoped styles once at the top of the file. That's why it took a constructor.

Reply:
Done.